content/en/service_management/incident_management/_index.md

```diff
@@ -114,32 +112,20 @@ For more information about the Datadog Slack integration, check out [the docs][7
 For {{< region-param key="dd_site_name" >}} customers who use Slack, stay informed about the Slack app by filing a ticket at https://help.datadoghq.com/.
 {{< /site-region >}}
 
-### Describing the incident
+## Describing the incident
 
 No matter where you create an incident, it's important to describe it as thoroughly as possible to share the information with other people involved in your company's incident management process.
 
 When you create an incident, an incident modal comes up. This modal has several core elements:
 
-**Severity Level**: Denotes the severity of your incident, from SEV-1 (most severe) to SEV-5 (least severe). If your incident is under initial investigation, and you do not know the severity yet, select UNKNOWN.
-
-* SEV-1: Critical impact
-* SEV-2: High impact
-* SEV-3: Moderate impact
-* SEV-4: Low impact
-* SEV-5: Minor issue
-* UNKNOWN: Initial investigation
-
-**Note**: You can customize the description of each severity level to fit the requirements of your organization.
-
-**Title**: Give your incident a descriptive title.
-
-**Signals**: The reason(s) you are declaring the incident. These can be graphs, logs, or other key visuals.
-
-**Incident commander**: This person is assigned as the leader of the incident investigation.
-
-**Additional notifications**: Notify other teams or people.
-
-Click on **Declare Incident** to finish creating your incident.
+| Incident elements    | Description |
+| ----------- | ----------- |
+| Title | (Required) Give your incident a descriptive title. |
+| Severity Level| (Required) Denotes the severity of your incident, from SEV-1 (most severe) to SEV-5 (least severe). If your incident is under initial investigation, and you do not know the severity yet, select UNKNOWN. <br> **Note**: You can customize the description of each severity level to fit the requirements of your organization.|
+| Incident Commander | This person is assigned as the leader of the incident investigation. |
+| Attributes (Teams) | Attach the appropriate group of users to an incident through [Datadog Teams][9], they are automatically invited to the Slack channels. |
```

Comment:
Should there be a word after the comma here? "Datadog Teams, they are"

Reply:
@drichards-87 Datadog Teams automatically invites the members of the assigned Team to the slack channels. Does this read better:
```suggestion
| Attributes (Teams) | Assign [Datadog Teams][9] with the appropriate group of users to an incident, this ensures the relevant Teams are automatically invited to the Slack channels. |
```

Reply:
How about something more along the lines of... "Assign the appropriate group of users to an incident using [Datadog Teams][9]. Members of the assigned team are automatically invited to the Slack channels."

Reply:
```suggestion
| Attributes (Teams) | Assign the appropriate group of users to an incident using [Datadog Teams][9]. Members of the assigned team are automatically invited to the Slack channels. |
```